src/index.ts

```diff
@@ -11,6 +11,10 @@ import type { Duplex, Source } from 'it-stream-types'
 import type { StreamMuxerFactory, StreamMuxerInit, StreamMuxer } from '@libp2p/interface-stream-muxer'
 import { Uint8ArrayList } from 'uint8arraylist'
 
+declare global {
+  var WebTransport: any
+}
+
```

Comment:
=(

Reply:
I'm not sure that we're ready to pull the `@fails-components/webtransport` dep in yet, we can revisit after https://github.com/libp2p/js-libp2p-webtransport/pull/31 or when webtransport gets added to the default ts types, whatever is faster.